services/director-v2/tests/unit/with_swarm/test_modules_dynamic_sidecar_docker_api.py

```diff
@@ -231,6 +232,36 @@ async def cleanup_dynamic_sidecar_stack(
         )
 
 
+@pytest.fixture
+async def project_id_labeled_network(
+    async_docker_client: aiodocker.docker.Docker, project_id: ProjectID
+) -> AsyncIterable[str]:
+    network_config = {
+        "Name": "test_network_by_project_id",
+        "Driver": "overlay",
+        "Labels": {"project_id": f"{project_id}"},
+    }
+    network_id = await docker_api.create_network(network_config)
+
+    yield network_id
+
+    network = await async_docker_client.networks.get(network_id)
+    await network.delete()
+
+
+@pytest.fixture
+async def test_networks(
+    async_docker_client: aiodocker.docker.Docker, docker_swarm: None
+) -> AsyncIterator[List[str]]:
+    network_names = [f"test_network_name__{k}" for k in range(5)]
+
+    yield network_names
+
+    for network_name in network_names:
+        docker_network = await async_docker_client.networks.get(network_name)
+        assert await docker_network.delete() is True
```

Comment:
same here

Reply:
This is sufficient. You get a reply form the docker engine if the network was removed.